### PR TITLE
(CONT-950) - Fix mismatched data types

### PR DESCRIPTION
**Pull request by jordanbreen28**
Fixes https://github.com/puppetlabs/puppetlabs-postgresql/issues/1426 
Fixes https://github.com/puppetlabs/puppetlabs-postgresql/issues/1423

### CODE REVIEW

File: manifests/server/config_entry.pp [hunk 1]
```diff
@@ -5,9 +5,9 @@
 # @param path Path for postgresql.conf
 #
 define postgresql::server::config_entry (
-  Enum['present', 'absent']             $ensure = 'present',
-  Optional[Variant[String[1], Integer]] $value  = undef,
-  Variant[Boolean, String[1]]           $path   = false
+  Enum['present', 'absent']                    $ensure = 'present',
+  Optional[Variant[String[1], Integer, Float]] $value  = undef,
```
Comment (ekohl) [inline manifests/server/config_entry.pp:9]: Elsewhere it was suggested that there's `Numeric` for both Integer and Float. https://www.puppet.com/docs/puppet/7/lang_data_abstract.html#lang_data_abstract_flexible-variant-data-type states `Variant[Integer, Float]` is equivalent to `Numeric`.

Comment (jordanbreen28) [reply manifests/server/config_entry.pp]: Thank you @ekohl, I like 👍

**Review by LukasAud — APPROVED** [review]
LGTM

File: manifests/server/config_entry.pp [hunk 2]
```diff
@@ -6,7 +6,7 @@
 #
 define postgresql::server::config_entry (
   Enum['present', 'absent']             $ensure = 'present',
-  Optional[Variant[String[1], Integer]] $value  = undef,
+  Optional[Variant[String[1], Numeric]] $value  = undef,
```
[the remark this answers is not in the document]
Comment (jordanbreen28) [reply manifests/server/config_entry.pp:9]: Doh! 
Opened https://github.com/puppetlabs/puppetlabs-postgresql/pull/1434 to resolve.
Thanks @ekohl

**Pull request merged**